<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdmChange</class>
 <widget class="QDialog" name="AdmChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frameAddUser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">visible:false
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lframeUSName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Добавление пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLogin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePass">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FIO">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFIO">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dom">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Дом</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDom">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Kvartira">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Квартира</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRoom">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Chel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во чел</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Priv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Привилегия №</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePriv">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameChangeUser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="frameChUsName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Редактирование пользователя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineLogin_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePass_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Login_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pass_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_FIO_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFIO_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dom_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>54</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Дом</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDom_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Kvartira_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Квартира</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRoom_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Chel_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во чел</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChel_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Priv_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Привилегия №</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePriv_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameChangePriv">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="framePrivName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Редактирование привелегий</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDecrease">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_decrease">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Уменьшение оплаты</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameChangePrice">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">visible:false
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="framePriceName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Редактирование цен</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linePrice">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Price">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Цена</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUnits">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Units">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Единицы измерения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPrice">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameChangeNews">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelNews">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите новость</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textNews">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPost">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Запостить</string>
    </property>
   </widget>
  </widget>
  <zorder>frameChangeUser</zorder>
  <zorder>buttonBox</zorder>
  <zorder>frameAddUser</zorder>
  <zorder>frameChangePrice</zorder>
  <zorder>frameChangePriv</zorder>
  <zorder>frameChangeNews</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdmChange</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdmChange</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
